packages/whatsie-git/whatsie-git.pacscript

```diff
@@ -0,0 +1,23 @@
+name="whatsie-git"
+gives="whatsie"
+arch=("any")
```

Comment:
```suggestion
```
If it works on everything, while "any" would *work*, removing it means the same.